sphinx/cmd/build.py

```diff
@@ -29,6 +29,11 @@
 
 
 def handle_exception(app: Sphinx, args: Any, exception: BaseException, stderr: IO = sys.stderr) -> None:  # NOQA
+    import bdb
```

Comment:
Why not importing this at the top of file?

Reply:
Thought this would be more in line with the lazy `import pdb` below.

I think `handle_exception` is worth importing lazily, since it is not expected to be used in the common case.

Reply:
IMO, it would be better to move all imports to the top of the script if no special reason (avoiding circular imports, becomes slow and so on). If lazy loading is needed for some special reason, adding comments will help us in the future.

Reply:
I'll fix this after merging.